src/seedsigner/models/settings.py

```diff
@@ -47,42 +47,22 @@ def save(self):
                 os.fsync(settings_file.fileno())
 
 
-    def update(self, new_settings: dict, disable_missing_entries: bool = True):
+    def update(self, new_settings: dict, ):
```

Comment:
Since this pr was submitted, pr-389 flow-tests framework has been merged.
In that pr, I was using the non-default "disabled_missing_entries=False" feature that is no longer present, so I'll be adjusting my test to setup in a one-by-one manner, otherwise this pr once merged would fail the test suite.

Reply:
Trailing comma fixed!

Reply:
it's a sneaky trailing comma.